_data/members.yaml

```diff
@@ -0,0 +1,19 @@
+---
+# to get a gravatar for a user:
+# just run:
+# echo "http://www.gravatar.com/avatar/$(echo "rsg1egoman@gmail.com" | md5sum | sed 's/  -//')"
+-
+- name: Ryan Gaus
+  img: http://rgaus.net/img/me.png
+  desc: Web Developer. I specialize in backend development. Teenager. Work for Lono.
+  social:
+    twitter: rgausnet
+    website: http://rgaus.net
+    email: rsg1egoman@gmail.com
+- name: John Doe
```

Comment:
Can you turn this john doe section into a commented out block at the top with descriptions for any fields that need it. (for instance the gravatar link, and I'd like to use liquid filters to cap the `desc` section to 100 chars. Be brief!)